### PR TITLE
[Java] Fix failed test

### PR DESCRIPTION
**Pull request by likholat**

### CODE REVIEW

File: modules/java_api/tests/InferRequestTests.java [hunk 1]
```diff
@@ -25,34 +24,34 @@ public void setUp() {
...
-    @Ignore
     @Test
     public void testGetPerformanceCounts() {
         inferRequest.Infer();
 
         Vector<String> layer_name = new Vector<>();
-        layer_name.add("19/Fused_Add_");
+        layer_name.add("19/WithoutBiases");
+        layer_name.add("20/mean/Fused_Mul_614616_const");
+        layer_name.add("20/mean/Fused_Mul_614616_const_abcd_Acdb16a_19/WithoutBiases");
```
Comment (dkurt) [inline modules/java_api/tests/InferRequestTests.java:34]: This is unstable check. The values are generated differently between runs and machines.

**Comment by dkurt** [comment]
@likholat, simplify the test. MO is unstable but we need to test API - check that map is not empty or something like that.

**Review by dkurt — APPROVED** [review]
